registries/princeton/documents.csv

```diff
@@ -7,3 +7,26 @@
 "PFS-PFI-PRU040001-01", "Cobra Movement Test Report June 2018", "Erin Kado-Fong", "https://sumire.pbworks.com/w/file/fetch/126883949/PFS-PFI-PRU040001-01_CobraTests_June2018.pdf", "2018-06-08", 
 "PFS-ICS-PRU030000-01", "PFS ICS Hardware Configuration Report", "Hassan Siddiqui", "https://sumire.pbworks.com/w/file/126994364/PFS-ICS-PRU030000-01_PFS_ICS_Hardware_Configuration_Report.pdf", "2018-05-07", 
 "PFS-DRP-PRU050001-01", "Interim report on the status of 2D PSF modelling", "Neven Caplar", "https://sumire.pbworks.com/w/file/fetch/127999977/PFS-DRP-PRU050001-01_2D_modelling_status.pdf", "2018-06-26",
+"PFS-GEN-PRU000010-01", "Uploading Whitepaper HOWTO", "Jim Gunn", "", "2011-01-01", "howto dummies"
+"PFS-PFI-PRU000020-01", "Summary of Subaru/WFC Imaging Performance", "Jim Gunn", "", "2012-04-04", "Summary of Subaru/WFC imaging performance for PFS with vigetting, angles, etc."
+"PFS-PFI-PRU000030-01", "Notes on rotator and A&G cameras", "Jim Gunn", "", "2011-12-03", "Notes on the rotator and A&G cameras and rotation limits"
+"PFS-PFI-PRU000040-01", "Notes on geometry of fiber slit", "Jim Gunn", "", "2012-01-07", "Notes on the geometry of the fiber slit"
+"PFS-PFI-PRU000050-01", "Notes on dither", "Jim Gunn", "", "2012-05-17", "Notes on the requirements for the dither mechanism for the FSA"
```

Comment:
Category : SPS
(This paper discusses "slit" dithering, not dithering on the sky.)